src/readii_2_roqc/visualization/plot_prediction.py

```
@@ -46,13 +53,19 @@ def build_prediction_df(dataset_config: dict,
 
         predictions[image_type] = image_type_predictions[evaluation_metric]
 
+    if predictions.empty:
+        message = f"No bootstrap predictions found at {bootstrap_predictions_path}. Check that the dataset and signature names are correct."
+        logger.error(message)
```

Ruff (0.14.1)

Undefined name `logger`

(F821)